webapp-django/crashstats/symbols/tests/test_views.py

```diff
@@ -67,6 +69,7 @@ def mocked_get_key(key_name):
                 if key_name in self.known_bucket_keys:
                     mocked_key = mock.Mock()
                     mocked_key.key = key_name
+                    mocked_key.content_type = 'application/binary-octet-stream'
```

Comment:
(for the benefit of anyone reading along, @peterbe figured out a good way to do this and I merged a PR in, github is not smart enough to collapse this though)